firefox/data/class_model.js

```
@@ -0,0 +1,278 @@
+    loadTrelloData() {
+        log("loading trello data");
+
+        this.event.fire("onBeforeLoadTrello");
+        this.trello.user = null;
+
+        // get user's info
+        log("Getting user info");
+        Trello.get("members/me", {}, (data) => {
+            this.trello.user = data;
+
+            if (!data || !data.hasOwnProperty("id")) return false;
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)